scripts/pcm-dpc/covid-19/dpc-covid19-ita_StatisticalVariable.mcf

```diff
@@ -0,0 +1,95 @@
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_IntensiveCare
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:IntensiveCare
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_Hospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:Hospitalized
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_PeopleInHomeIsolation
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PeopleInHomeIsolation
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_ActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:ActiveCase
+
+Node: dcid:incrementalCount_MedicalConditionIncident_COVID_19_IncrementalActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:IncrementalActiveCase
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_Recovered
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:Recovered
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_Death
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:Death
+
+Node: dcid:cumulativeCount_MedicalConditionIncident_COVID_19_PositiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:PositiveCase
+
+Node: dcid:incrementalCount_MedicalConditionIncident_COVID_19_IncrementalPositiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:IncrementalPositiveCase
+
+Node: dcid:cumulativeCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicaltTest
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+
+Node: dcid:cumulativeCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:Tested
+statType: dcs:measuredValue
+
```

Comment:
Would generally be good to go thru https://github.com/datacommonsorg/data/blob/master/scripts/covid_tracking_project/historic_state_data/COVIDTracking_States_StatisticalVariables.mcf to see if any others can be reused (in case I missed one)

Reply:
Another feedback is the StatVar IDs should be PascalCased

Reply:
Got it.